classes/class-blog-creator-factory.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+namespace EdLTI;
```

Comment:
Name spaces should reflect the directory.
```suggestion
namespace EdLTI\classes;
```